src/FormGroup/FormGroup.jsx

```diff
@@ -118,7 +118,7 @@ FormGroup.propTypes = {
   displayErrorText: PropTypes.bool,
   elementType: PropTypes.oneOf(FORM_GROUP_ELEMENT_TYPES),
   errors: PropTypes.object,
-  helperText: PropTypes.string,
+  helperText: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
```

Comment:
Technically, a string is also a node, so this can likely just be:
```suggestion
  helperText: PropTypes.oneOfType([PropTypes.node]),
```

Reply:
Might as well do `helperText: PropTypes.node`

Reply:
ah yea!